lib/data_update_scripts/20200910140109_cleanup_published_articles_with_duplicate_user_id_title_body_markdown.rb

```diff
@@ -0,0 +1,104 @@
+module DataUpdateScripts
+  class CleanupPublishedArticlesWithDuplicateUserIdTitleBodyMarkdown
```

Comment:
In an internal discussion on Slack we've decided to go ahead and "merge" duplicate articles

cc @mstruve